src/common/TokenParser.ts

```diff
@@ -134,6 +139,7 @@ export class TokenParser {
             totalSupply: obj.totalSupply,
             decimals: obj.decimals,
             symbol: obj.symbol,
+            verified: obj.verified
```

Comment:
would it be possible to store this value in the mongoldb? we would need this to sort then and order by verified first